web/src/components/layout/footer.tsx

```diff
@@ -15,8 +15,11 @@ import './footer.css';
 const LocalizedLocaleLink = ({ id, to }: { id: string; to: string }) => {
   const [locale] = useLocale();
   return (
-    <Localized id={id} onClick={() => trackNav(id, locale)}>
```

Comment:
`Localized` doesn't render any own DOM, nor does it pass props down to children directly. The `onClick` here probably never worked.